VRC.Car.Pi/hardware_controller.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import threading
+from time import sleep
+from ctypes import c_char
+import board
+import busio
+import i2c_constants
+
+class HardwareController:
+
+  _i2c_lock = threading.RLock()
+  _i2c = busio.I2C(board.SCL, board.SDA)
+  _atmega1 = 0x20
+  _atmega2 = 0x30
+
+  def __init__(self):
+    print("Initializing devices...")
+    self.test_i2c_devices()
+
+  def test_i2c_devices(self):
+    print("Testing I2C devices...")
+    bytesToSend = bytearray([32])
+    try:
+      self._i2c_lock.acquire()
+      # self._i2c.writeto(self._atmega1, bytesToSend)
+      self._i2c_lock.release()
+    except:
+      sys.exit("Problem connecting to atmega1")
+
+    try:
+      self._i2c_lock.acquire()
+      # self._i2c.writeto(self._atmega2, bytesToSend)
```

Comment:
moet deze commentaar blijven staan of is die vergeten weg te halen?

Reply:
er is geen atmega 2 momenteel